<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FOCUS-QA-118" name="com.focus.testcases.UserRestrictionsTest" tests="22" failures="3" timestamp="28 Dec 2022 08:56:33 GMT" time="550.391" errors="0">
  <testcase name="checkReceiptsHomePageMasterList" time="3.876" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkDepartmentMasterWhetherShowingOnlyRestrictedDepartmentInList" time="16.043" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkPurchasesVoucherForAllTheMastersListInMastersDropdown" time="26.674" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkSalesInvoiceVoucherHomePgaeMastersList" time="9.695" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkPurchaseVoucherHomePageMastersList" time="12.773" classname="com.focus.testcases.UserRestrictionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.UserRestrictionsTest.checkPurchaseVoucherHomePageMastersList(UserRestrictionsTest.java:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkPurchaseVoucherHomePageMastersList -->
  <testcase name="checkItemMasterWhetherShowingOnlyRestrictedItemsInList" time="18.044" classname="com.focus.testcases.UserRestrictionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.UserRestrictionsTest.checkItemMasterWhetherShowingOnlyRestrictedItemsInList(UserRestrictionsTest.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkItemMasterWhetherShowingOnlyRestrictedItemsInList -->
  <testcase name="checkLogin" time="114.323" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkRestoreCompany" time="40.636" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkAccountMasterWhetherShowingOnlyRestrictedAccountsInList" time="7.172" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkBranchMasterWhetherShowingOnlyRestrictedBranchInList" time="8.017" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkSearchItemsWhetherNonRestrictedItemsAreDisplaying" time="57.643" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkMastersListInPayments" time="16.879" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkChildAccountsInsideTheGroupAccountsIndivdually" time="25.325" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkMastersListInSalesInvoices" time="20.794" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkMasterInReceipts" time="13.661" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkLoginWithERR" time="15.191" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkValidateRestrictionsGivenInRERRoleInSu" time="16.737" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkPaymentsVoucherHomePageMastersList" time="4.592" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkWarehouseMasterWhetherShowingOnlyRestrictedWarehouseInList" time="15.990" classname="com.focus.testcases.UserRestrictionsTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at com.focus.testcases.UserRestrictionsTest.checkWarehouseMasterWhetherShowingOnlyRestrictedWarehouseInList(UserRestrictionsTest.java:103)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- checkWarehouseMasterWhetherShowingOnlyRestrictedWarehouseInList -->
  <testcase name="checkSearchDepartmentWhetherNonRestrictedDepartmentsAreDisplaying" time="47.208" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkSearchAccountsWhetherNonRestrictedAccountsAreDisplaying" time="12.730" classname="com.focus.testcases.UserRestrictionsTest"/>
  <testcase name="checkSearchWarehouseWhetherNonRestrictedWarehousesAreDisplaying" time="46.388" classname="com.focus.testcases.UserRestrictionsTest"/>
</testsuite> <!-- com.focus.testcases.UserRestrictionsTest -->
